apps/faucet/src/App/FaqDropdown.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import { FaqDropdownContainer, FaqDropdownContent, PlusIcon, DropDownTitle, DropDownTitleText } from "./Faq.components";
+import plusIcon from '../../public/plus-icon.svg'
+
+type Props = {
+  title: string;
+  children: React.ReactNode;
+};
+export const FaqDropdown: React.FC<Props> = ({ children, title }) => {
+  const [isOpen, setIsOpen] = useState<boolean | null>(null);
+
+  const handleToggle = (): void => {
+    setIsOpen((prevIsOpen) => !prevIsOpen);
+  };
+
+  return (
+    <FaqDropdownContainer onClick={handleToggle}>
+      <DropDownTitle>
+        <DropDownTitleText>
+          {title}
+        </DropDownTitleText>
+        <PlusIcon src={plusIcon} isOpen={isOpen} />
+      </DropDownTitle>
+
+      <FaqDropdownContent isOpen={isOpen}>{children}</FaqDropdownContent>
+       
```

Comment:
There is a trailing whitespace error in this line. (Actually, this should have been caught by the linter so I'm a bit surprised it wasn't, but I will fix the linting config in a separate PR.)

Reply:
Strange. Run lint no more issues were detected 
Also, it would be great to add eslint to dev dependency. Because if you haven't eslint  globally you have to run something like that 
`npx eslint src --ext .ts,.tsx`

Reply:
Created PR example with a solution: https://github.com/anoma/namada-interface/pull/505